sdks/js/packages/bare/src/index.ts

```diff
@@ -20,27 +12,5 @@ Object.defineProperty(Module, "_resolveFilename", {
   configurable: false,
 });
 
-globalThis.Intl = {
-  NumberFormat: () => {
-    return {
-      resolvedOptions: () => ({
-        locale: "en-US",
-        numberingSystem: "latn",
-        style: "decimal",
-        minimumIntegerDigits: 1,
-        minimumFractionDigits: 0,
-        maximumFractionDigits: 3,
-        useGrouping: "auto",
-        notation: "standard",
-        signDisplay: "auto",
-        roundingIncrement: 1,
-        roundingMode: "halfExpand",
-        roundingPriority: "auto",
-        trailingZeroDisplay: "auto",
-      }),
-    };
-  },
-} as unknown as typeof Intl;
```

Comment:
Is this patched in bare-node-runtime now?

Reply:
it is shipped with `bare` so you won't see it in `bare-node-runtime`. please update your bare version to v1.23.4